src/redux/sagas/actions/createDomain.ts

```diff
@@ -166,7 +166,7 @@ function* createDomainAction({
     });
 
     if (colonyName && navigate) {
-      navigate(`/colony/${colonyName}/tx/${txHash}`, {
+      navigate(`/${colonyName}?tx=${txHash}`, {
```

Comment:
These conditional blocks should be removed when we no longer need to initiate actions/motions from the old ui. But updating anyway

Reply:
Leave a comment similar to what I have here: https://github.com/JoinColony/colonyCDapp/blob/master/src/routes/Routes.tsx#L184